insilicho/parameters.py

```diff
@@ -68,6 +68,12 @@ class InputParameters(UnitValidationMixin):
     Cglc_feed: typing.Union[float, str] = 150.0  # mmol/liter
     Cgln_feed: typing.Union[float, str] = 10.0  # mmol/liter
 
+    # Optimal
+    T_optimal: float = 36.40
```

Comment:
why not `typing.Union[float, str]`? and should this also be subject to noise?

also would be good to have units in comments

Reply:
i dont think it should be subject to noise but not sure. Can exclude for now manually?